modules/HapticGlove_module/src/LinearRegression.cpp

```diff
@@ -0,0 +1,61 @@
+#include <LinearRegression.hpp>
+
+using namespace HapticGlove;
+
+LinearRegression::LinearRegression()
+{
+}
+
+LinearRegression::~LinearRegression()
+{
+}
+
+bool LinearRegression::Initialize()
+{
+
+    return true;
+}
+
+bool LinearRegression::LearnOneShot(const CtrlHelper::Eigen_Mat& inputData,
+                                    const CtrlHelper::Eigen_Mat& ouputData,
+                                    CtrlHelper::Eigen_Mat& tetha)
+{
+
+    // TODEL
+    std::cout << "[LinearRegression::LearnOneShot()]" << std::endl;
+
+    CtrlHelper::Eigen_Mat xT_x = (inputData.transpose() * inputData);
+    std::cout << "xT_x: " << xT_x.rows() << xT_x.cols() << std::endl;
+    std::cout << "xT_x.determinant(): " << xT_x.determinant() << std::endl;
```

Comment:
The computation of the determinant might be expensive. Not sure if it is worth for a simple debug message